lib/engine-addon.js

```diff
@@ -362,6 +362,18 @@ module.exports = {
       }).filter(Boolean);
     };
 
+    /**
+     * Determines if ember-engines should use the compact-reexports Babel plugin.
+     */
+    options.shouldCompactReExports = function() {
+      var VersionChecker = require('ember-cli-version-checker');
+      var checker = new VersionChecker(this);
+
+      return checker.for('ember-cli', 'npm').satisfies('>=2.13.0') &&
+        checker.for('ember-cli-babel', 'npm').satisfies('>=6.0.0') &&
```

Comment:
Checking for ember-cli and loader.js here is fine, because those are actually project level deps.